README.md

```diff
@@ -58,15 +58,11 @@ These three libraries provide a powerful combination of HPC message passing tool
 
     git clone https://github.com/openucx/ucx
     cd ucx
-    git remote add Akshay-Venkatesh git@github.com:Akshay-Venkatesh/ucx.git
-    git remote update Akshay-Venkatesh
-    git checkout ucx-cuda
     ./autogen.sh
     mkdir build
     cd build
     ../configure --prefix=$CONDA_PREFIX --enable-debug --with-cuda=$CUDA_HOME --enable-mt --disable-cma CPPFLAGS="-I//$CUDA_HOME/include"
```

Comment:
I think we can drop `--disable-cma` now as that was fixed in `master`. ( https://github.com/openucx/ucx/pull/3424 )